### PR TITLE
[bitnami/thanos] fix: add tls switch to enable TLS with cert-manager for querier and bucketweb ingress

### LINKED ISSUE #4127: [bitnami/thanos] querier and bucketweb ingress config
**Which chart**:
Thanos, after update to version 2.4.0

**Describe the bug**
Not able to create querier and bucketweb ingress with tls config - [code](https://github.com/bitnami/charts/blob/master/bitnami/thanos/templates/querier/ingress.yaml#L34:L39)

**To Reproduce**
My config
```
querier:
  ingress:
    enabled: true
    hostname: thanos-${name}.${operational_dns_name}
    annotations:
      kubernetes.io/ingress.class: operational
      cert-manager.io/cluster-issuer: "letsencrypt-production"
```

**Expected behavior**
Creates ingress object with tls configuration block

**Version of Helm and Kubernetes**:

- Output of `helm version`: 

```
Applying through ArgoCD 1.7.8
```

**Additional context**
kube-prometheus-stack is using this configuration. How should I configure this with this chart?
```
tls:
  - hosts:
      - alertmanager-${name}.${operational_dns_name}
    secretName: alertmanager-${name}.${operational_dns_name}-tls
```

### PR DESCRIPTION
**Pull request by janlud**
The querier.ingress.tls conditional was removed in previous releases which enabled cert-manager to work with `cert-manager.io/cluster-issuer: nameOfClusterIssuer` annotation.

**Description of the change**
Add back a `querier.ingress.tls` and `bucketweb.ingress.tls`which was removed. This switch is important for cert-manager provisioning of certificates.
When `tls: true` a tls: part of ingress manifest is rendered. For this also `cert-manager.io/cluster-issuer: nameOfClusterIssuer` annotation must be set. That is already supported by using annotations
Example of what should be created is [here](https://cert-manager.io/docs/usage/ingress/#how-it-works)

**Benefits**

Enabled use of cert-manager cert provisioning for querier and bucketweb ingresses.

**Possible drawbacks**

**Applicable issues**

  - fixes #4127

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/thanos/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 2.5.3
+version: 2.5.4
```
Comment (miguelaeh) [inline bitnami/thanos/Chart.yaml:2]: You are adding a new parameter this should be a minor bump

File: bitnami/thanos/values-production.yaml [hunk 2]
```diff
@@ -792,6 +796,10 @@ bucketweb:
     ## It is also possible to create and manage the certificates outside of this helm chart
     ## Please see README.md for more information
     ##
+    ## When specifying cert-manager.io/cluster-issuer: nameOfClusterIssuer annotation, enable tls for ingress
```
Comment (miguelaeh) [inline bitnami/thanos/values-production.yaml:799]: I would separate this from the previous section. Please add a space before, and the same in the rest of the changes like this one

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!
Please take a look at my comments

**Comment by janlud** [comment]
@miguelaeh I made the required changes. Made some mess in git history and took me a while to figure it out (sorry, still learning!). I see I have commits here from my other change I did couple days ago to thanos chart but I don't see those changes in diff (The PR was merged so it's in master). Can you merge it like this with squashing commits or do I need to somehow get those commits out?

**Review by dani8art — APPROVED** [review]
LGTM

**Pull request merged**